release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -0,0 +1,1538 @@
+
+Commencing with the {product-version} release, Red Hat is simplifying the administration and management of Red Hat shipped cluster Operators with the introduction of three new life cycle classifications; Platform Aligned, Platform Agnostic, and Rolling Stream. These life cycle classifications provide additional ease and transparency for cluster administrators to understand the life cycle policies of each Operator and form cluster maintenance and upgrade plans with predictable support boundaries. For more information, see link:https://access.redhat.com/webassets/avalon/j/includes/session/scribe/?redirectTo=https%3A%2F%2Faccess.redhat.com%2Fsupport%2Fpolicy%2Fupdates%2Fopenshift_operators[OpenShift Operator Life Cycles].
+
+// Added in 4.14. Language came directly from Kirsten Newcomer.
+{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the `x86_64`, `ppc64le`, and `s390x` architectures.
+
+For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of {op-system-base} cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].
+
+[id="ocp-4-15-add-on-support-status"]
+== {product-title} layered and dependent component support and compatibility
+
+The scope of support for layered and dependent components of {product-title} changes independently of the {product-title} version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+
+[id="ocp-4-15-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[id="ocp-4-15-rhcos"]
+=== {op-system-first}
+
+[id="ocp-4-15-installation-and-update"]
+=== Installation and update
+
+[id="ocp-4-15-web-console"]
+=== Web console
+
+[id="ocp-4-15-openshift-cli"]
+=== OpenShift CLI (oc)
+
+[id="ocp-4-15-ibm-z"]
+=== {ibmzProductName} and {linuxoneProductName}
+
+[id="ocp-4-15-ibm-power"]
+=== {ibmpowerRegProductName}
+
+[discrete]
+==== {ibmpowerRegProductName} notable enhancements
+
+[discrete]
+=== {ibmpowerRegProductName}, {ibmzRegProductName}, and {linuxoneProductName} support matrix
+
+.{product-title} features
```

Comment:
Something I thought about was whether or not these tables could be moved to the main docs and then simply referenced here because they take up a lot of space. I don't know though if that works with the user story for these tables.

Reply:
Probably don't want to do that this time but I'll keep it in mind.

Reply:
A good iterative step could be to modularize them and add them to the RN assembly. Not suggesting that in this PR (I think it's good to cut the 4.15 RNs just like 4.14 before diverging) but maybe later